src/app/Http/Controllers/VolunteerController.php

```diff
@@ -240,15 +242,23 @@ public function store(Request $request) {
         if ($request->has('county')) {
             $data['county'] = getCityOrCounty($request->county,County::query());
         }
-        if ($request->has('city')) {            
+        if ($request->has('city')) {
             $data['city'] = getCityOrCounty($request->city,City::query());
         }
 
         /** Add 'adden by' to the the volunteer */
         $data['added_by'] = \Auth::check() ? \Auth::user()->_id : '';
         $data['courses'] = [];
-        /** Create the volunteer. */
-        $volunteer = Volunteer::create($data);
+
+        /** if SSN already exists in database then update the organisation_id and course */
+        $volunteer = Volunteer::query()->where('ssn', '=', $data['ssn'])->first();
+        if($volunteer){
+            $volunteer->organisation = $data['organisation'];
+            $data['courses'] = $volunteer->courses;
+        }else{
```

Comment:
Was this behavior intended?
This means that a volunteer could be moved from organization A to organization B without informing organization A of this change.

Reply:
The requirement for this ticket was: "When a volunteer is added, a database search is performed in the volunteer table by CNP. If the volunteer exists, the organization_id column and the courses column are updated.". Nothing about notifies!

Reply:
You are right.